test/test_csvs.py

```diff
@@ -0,0 +1,291 @@
+# OOIPLACEHOLDER
+#
+# Copyright 2018 Raytheon Co.
+
+import io
+import os
+import unittest
+from nose.plugins.attrib import attr
+
+@attr('UNIT')
+class GeneralUnitTest(unittest.TestCase):
+    # declare the global validation objects
+    _refdes = set()
+    _units = set()
+    _parameters = {}
+    _types = {}
+
+    def _getpath(self, filename):
+        return os.path.join(os.path.dirname(__file__), filename)
+    
+    def setUp(self):
+        # initialize the global objects once
+        if not self._refdes:
+            with io.open(self._getpath('refdes.txt'),mode='rt',encoding='iso-8859-1') as f:
```

Comment:
@tlfish3638 I suggest revising the code to work against the latest version of the csv files. specifying the 'iso-8859-1' encoding is probably no longer necessary.

Reply:
I'll update this file to uniformly use utf-8 encoding instead of iso-8859-1.

Reply:
Again, please DO NOT use shell scripting to parse data files. Python is a much safer and efficient tool for doing this.